proxmoxtf/resource_virtual_environment_container.go

```diff
@@ -1396,12 +1420,18 @@ func resourceVirtualEnvironmentContainerRead(ctx context.Context, d *schema.Reso
 			err := d.Set(mkResourceVirtualEnvironmentContainerDiskDatastoreID, []interface{}{disk})
 			diags = append(diags, diag.FromErr(err)...)
 		}
-	} else if len(currentDisk) > 0 ||
-		disk[mkResourceVirtualEnvironmentContainerDiskDatastoreID] != dvResourceVirtualEnvironmentContainerDiskDatastoreID {
-		err := d.Set(mkResourceVirtualEnvironmentContainerDiskDatastoreID, []interface{}{disk})
-		diags = append(diags, diag.FromErr(err)...)
 	}
 
+	/*
+		I can't use disk datastore otherthan local-lvm and it cause the error
+		"Error: Invalid address to set: []string{"datastore_id"}"
+	*/
+	// } else if len(currentDisk) > 0 ||
+	// 	disk[mkResourceVirtualEnvironmentContainerDiskDatastoreID] != dvResourceVirtualEnvironmentContainerDiskDatastoreID {
+	// 	err := d.Set(mkResourceVirtualEnvironmentContainerDiskDatastoreID, []interface{}{disk})
+	// 	diags = append(diags, diag.FromErr(err)...)
+	// }
```

Comment:
@abdo-farag I pushed the fix, could you rebase and uncomment this block to see if it works for you?

Reply:
@bpg It works!, Thanks

Reply:
> @bpg It works!, Thanks

Great! Could you please put the SSD work in a separate PR, and we'll focus here on MTU?
Thanks!

Reply:
@bpg I have created new branch for ssd feature is it better so? I don't know how to split them :)